FreeOTP/ShareViewController.swift

```diff
@@ -215,7 +215,7 @@ class ShareViewController : UITableViewController, CBCentralManagerDelegate, CBP
             central.scanForPeripherals(withServices: [SERVICE], options: nil)
 
         default:
-            central.stopScan()
+            break
```

Comment:
I don't understand this code change. Can you clarify?

Reply:
CoreBluetooth was logging errors about the state machine; I'll double check this change in the revised request.